Core/DISET/private/ServiceConfiguration.py

```diff
@@ -1,110 +1,115 @@
+      self.pathList.append(PathFinder.getServiceSection(svcName))
 
-  def getOption( self, optionName ):
+  def getOption(self, optionName):
     if optionName[0] == "/":
-      return gConfigurationData.extractOptionFromCFG( optionName )
+      return gConfigurationData.extractOptionFromCFG(optionName)
     for path in self.pathList:
-      value = gConfigurationData.extractOptionFromCFG( "%s/%s" % ( path, optionName ) )
+      value = gConfigurationData.extractOptionFromCFG("%s/%s" % (path, optionName))
       if value:
         return value
     return None
 
-  def getAddress( self ):
-    return ( "", self.getPort() )
+  def getAddress(self):
+    return ("", self.getPort())
 
-  def getHandlerLocation( self ):
-    return self.getOption( "HandlerPath" )
+  def getHandlerLocation(self):
+    return self.getOption("HandlerPath")
 
-  def getName( self ):
+  def getName(self):
     return self.serviceName
 
-  def setURL( self, sURL ):
+  def setURL(self, sURL):
     self.serviceURL = sURL
 
-  def __getCSURL( self, URL = None ):
-    optionValue = self.getOption( "URL" )
+  def __getCSURL(self, URL=None):
+    optionValue = self.getOption("URL")
     if optionValue:
       return optionValue
     return URL
 
-  def registerAlsoAs( self ):
-    optionValue = self.getOption( "RegisterAlsoAs" )
+  def registerAlsoAs(self):
+    optionValue = self.getOption("RegisterAlsoAs")
     if optionValue:
-      return List.fromChar( optionValue )
+      return List.fromChar(optionValue)
     else:
       return []
 
-  def getMaxThreads( self ):
+  def getMaxThreads(self):
     try:
-      return int( self.getOption( "MaxThreads" ) )
+      return int(self.getOption("MaxThreads"))
     except:
       return 15
 
-  def getMinThreads( self ):
+  def getMinThreads(self):
     try:
-      return int( self.getOption( "MinThreads" ) )
+      return int(self.getOption("MinThreads"))
     except:
       return 1
 
-  def getMaxWaitingPetitions( self ):
+  def getMaxWaitingPetitions(self):
     try:
-      return int( self.getOption( "MaxWaitingPetitions" ) )
+      return int(self.getOption("MaxWaitingPetitions"))
     except:
       return 500
 
-  def getMaxMessagingConnections( self ):
+  def getMaxMessagingConnections(self):
     try:
-      return int( self.getOption( "MaxMessagingConnections" ) )
+      return int(self.getOption("MaxMessagingConnections"))
     except:
       return 20
 
-  def getMaxThreadsForMethod( self, actionType, method ):
+  def getMaxThreadsForMethod(self, actionType, method):
     try:
-      return int( self.getOption( "ThreadLimit/%s/%s" % ( actionType, method ) ) )
+      return int(self.getOption("ThreadLimit/%s/%s" % (actionType, method)))
     except:
       return 15
 
-  def getCloneProcesses( self ):
+  def getCloneProcesses(self):
     try:
-      return int( self.getOption( "CloneProcesses" ) )
+      return int(self.getOption("CloneProcesses"))
     except:
-      return 1
+      return 0
```

Comment:
Why?

Reply:
This is the number of how many processes needs to be started. By default we do not want to start any processes